kvbc/include/base_db_adapter.h

```diff
@@ -23,18 +23,9 @@ class DBAdapterBase {
     db_->init(false);
   }
 
- public:
-  std::shared_ptr<storage::IDBClient> getDb() const { return db_; }
-
-  storage::IDBClient::IDBClientIterator *getIterator() { return db_->getIterator(); }
-
-  concordUtils::Status freeIterator(storage::IDBClient::IDBClientIterator *_iter) { return db_->freeIterator(_iter); }
-
  protected:
```

Comment:
Do we still need this class at all?

Reply:
Good point. I don't think so, removing it.